### ISSUE TITLE
[错误]: Qwen3系列模型的推理过程不显示

### ISSUE THREAD

**Issue by amantuer** [CherryHQ/cherry-studio#7085]
### 提交前检查

- [x] 我理解 Issue 是用于反馈和解决问题的，而非吐槽评论区，将尽可能提供更多信息帮助问题解决。
- [x] 我的问题不是 [常见问题](https://github.com/CherryHQ/cherry-studio/issues/3881) 中的内容。
- [x] 我已经查看了 **置顶 Issue** 并搜索了现有的 [开放Issue](https://github.com/CherryHQ/cherry-studio/issues)和[已关闭Issue](https://github.com/CherryHQ/cherry-studio/issues?q=is%3Aissue%20state%3Aclosed%20)，没有找到类似的问题。
- [x] 我填写了简短且清晰明确的标题，以便开发者在翻阅 Issue 列表时能快速确定大致问题。而不是“一个建议”、“卡住了”等。

### 平台

Windows

### 版本

v1.4.1

### 错误描述

Qwen3系列模型的推理过程不显示，一般只显示推理使用了1.2秒之类的然后回复就开始了。看处理的时间感觉好像没有触发推理。有两点：(1).其他推理模型可以正常显示推理过程。 (2).同一个Qwen3模型使用别的前端(AnythingLLM, OpenWebUi)都可以正常显示推理过程。

### 重现步骤

使用unsloth的Qwen3-32B-128k, 30B-A22B-128k的Q_8版本, 后端使用LLM Studio

### 预期行为

正常显示Qwen3系列模型的推理过程

### 相关日志输出

```shell

```

**Comment by EurFelux** [comment]
我测试阿里云的qwen3推理没什么问题。你能提供更多上下文信息吗？比如你期望推理却没有正常推理的消息，看看其请求体和响应。

> 一般只显示推理使用了1.2秒之类的然后回复就开始了

意思是深度思考的ui会显示一会儿，然后消失了？还是说有深度思考的内容在里面？

**Comment by alephpiece** [comment]
可能是因为返回了空的 think tag

**Comment by amantuer** [comment]
@EurFelux 深度思考的ui会出来一秒左右，并且它只会显示“思考用了1.2秒”之类的时间消耗的情况，然后回复就会开始，没有深度思考的内容在里面。正常的思考模型的回复响应过程应该是：深度思考的ui出来以后先显示思考的过程“大量的文字，一般几百个字，持续一分钟到两分钟左右”然后再显示“思考用了59.1秒”之类的时间消耗的情况，然后在思考ui的下面回复再开始。   

你可以尝试在后端LLMStudio上运行这个模型: https://huggingface.co/unsloth/Qwen3-32B-128K-GGUF (我运行的Q_8, context window 拉满了131k左右，Evaluation batch size是16384， 其他选项默认没有变， 一般其他思考模型也是一样的选择)， unsloth也是一个受欢迎的权威的微调模型提供商，它家的模型质量一直很不错。

任何消息都没有推理，我发现LLMStudio接受到的请求体里面enable thinking是false, 但是cherry-studio里面在该模型的设置里默认推理是打开的。下面是LLMStudio接受到的请求体的具体内容(由于没有进行推理在下面这个请求中我在模型回复开始后手动结束了回复过程)：

2025-06-12 10:11:31 [DEBUG]
Received request: POST to /v1/chat/completions with body  {
  "model": "qwen3-32b-128k",
  "messages": [
    {
      "role": "system",
      "content": "请启用推理功能，用力思考，完美的完成用户要求的所有任务。"
    },
    {
      "role": "user",
      "content": "介绍一个高级KKT方程式然后做一个全面细致的解。 /no_think"
    }
  ],
  "keep_alive": "0m",
  "stream": true,
  "enable_thinking": false
}
2025-06-12 10:11:31  [INFO]
[LM STUDIO SERVER] Running chat completion on conversation with 2 messages.
2025-06-12 10:11:31  [INFO]
[LM STUDIO SERVER] Streaming response...
2025-06-12 10:11:31 [DEBUG]
Sampling params:
2025-06-12 10:11:31 [DEBUG]
repeat_last_n = 64, repeat_penalty = 1.100, frequency_penalty = 0.000, presence_penalty = 0.000
dry_multiplier = 0.000, dry_base = 1.750, dry_allowed_length = 2, dry_penalty_last_n = -1
top_k = 40, top_p = 0.950, min_p = 0.050, xtc_probability = 0.000, xtc_threshold = 0.100, typical_p = 1.000, top_n_sigma = -1.000, temp = 0.800
mirostat = 0, mirostat_lr = 0.100, mirostat_ent = 5.000
sampling:
logits -> logit-bias -> penalties -> dry -> top-n-sigma -> top-k -> typical -> top-p -> min-p -> xtc -> temp-ext -> dist
generate: n_ctx = 131072, n_batch = 16384, n_predict = -1, n_keep = 46
Detected 2/913 cached tokens to be a valid prefix of the current prompt. Removing 911 tokens from the end of the cache
2025-06-12 10:11:31 [DEBUG]
BeginProcessingPrompt
2025-06-12 10:11:32 [DEBUG]
FinishedProcessingPrompt. Progress: 100
2025-06-12 10:11:32  [INFO]
[LM STUDIO SERVER] First token generated. Continuing to stream response..
2025-06-12 10:13:47  [INFO]
[LM STUDIO SERVER] Client disconnected. Stopping generation... (If the model is busy processing the prompt, it will finish first.)
2025-06-12 10:13:47 [DEBUG]
target model llama_perf stats:
llama_perf_context_print:        load time =   22711.03 ms
llama_perf_context_print: prompt eval time =   18759.32 ms /   957 tokens (   19.60 ms per token,    51.01 tokens per second)
llama_perf_context_print:        eval time =  134488.66 ms /  1301 runs   (  103.37 ms per token,     9.67 tokens per second)
llama_perf_context_print:       total time =  136525.57 ms /  2258 tokens
2025-06-12 10:13:47  [INFO]
Finished streaming response

然后我发现如果在发送消息时在聊天框点开了"思维链长度"中的“沉思”选项，它会开始有推理过程的回复。但是别的思考模型比如"Qwq32B"你在发消息时不点开任何"思维链长度"中的选项它也是自动带有推理过程的回复。

**Comment by purefkh** [comment]
> 然后我发现如果在发送消息时在聊天框点开了"思维链长度"中的“沉思”选项，它会开始有推理过程的回复。但是别的思考模型比如"Qwq32B"你在发消息时不点开任何"思维链长度"中的选项它也是自动带有推理过程的回复。

看这段描述这是正常行为。默认思维链长度是关闭，所以 qwen3 没有输出 cot。而 qwq32b 应该不支持关闭思考。

**Comment by amantuer** [comment]
@purefkh 但是别的前端例如OpenWebUI, AnythingLLM中没有这个问题，难道说他们默认Qwen3的思维链默认是开启的吗？我没有在这些前端发现相关的设置选项。在ChatWise里面也没有问题，Qwen3是自动带推理回复的。

**Comment by EurFelux** [comment]
> [@purefkh](https://github.com/purefkh) 但是别的前端例如OpenWebUI, AnythingLLM中没有这个问题，难道说他们默认Qwen3的思维链默认是开启的吗？我没有在这些前端发现相关的设置选项。在ChatWise里面也没有问题，Qwen3是自动带推理回复的。

是的，开源版qwen3默认开启思考，但Cherry Studio默认关闭思考，所以会覆盖掉。
